go.mod

```diff
@@ -43,10 +43,10 @@ require (
 	github.com/googleapis/gax-go/v2 v2.9.1
 	github.com/gorilla/handlers v1.5.1
 	github.com/hashicorp/go-hclog v1.5.0
-	github.com/hashicorp/go-plugin v1.4.6
-	github.com/hashicorp/hcl v1.0.1-0.20190430135223-99e2f22d1c94
+	github.com/hashicorp/go-plugin v1.4.8
```

Comment:
We can't update go-plugin until there is a release with the fix for https://github.com/hashicorp/go-plugin/issues/232 (v1.4.9 doesn't include this fix either)